message/message.go

```diff
@@ -0,0 +1,20 @@
+package message
+
+// Message represents a message flowing through the wire. For example, a Kafka message.
+type Message struct {
+	Context Context  `json:"context"`
+	Key     []byte   `json:"key,omitempty"`
+	Value   []byte   `json:"value,omitempty"`
```

Comment:
Is this the same message as in AsyncAPI? I am missing the `payload` field. Unless `payload` is a `value`.

Reply:
No, this is not related to AsyncAPI at all. This is only a representation of message flowing through somewhere, E.g. A Kafka message flowing through a topic partition. But could be also any other protocol.

Reply:
AsyncAPI Models are under https://github.com/asyncapi/event-gateway/tree/master/asyncapi but weren't changed on this PR.

Reply:
Ok, thanks! 😄